bootkube-up

```diff
@@ -219,7 +219,7 @@ sudo chmod 644 ~/.kube/config
 
 
 echo_green "\nPhase IX: Running Bootkube start to bring up the temporary Kubernetes self-hosted control plane:"
-nohup sudo bash -c 'bootkube start --asset-dir='$BOOTKUBE_DIR'/.bootkube' >$BOOTKUBE_DIR/bootkube-ci/log/bootkube-start.log 2>&1 &
+nohup sudo bash -c 'bootkube start --asset-dir=$BOOTKUBE_DIR'/.bootkube' >$BOOTKUBE_DIR/bootkube-ci/log/bootkube-start.log 2>&1 &
```

Comment:
```
'bootkube start --asset-dir='$BOOTKUBE_DIR'/.bootkube'
```

there are two pairs of single-tick quotes in original command. what you submitted has an odd number.